common-test/src/main/java/io/customer/commontest/BaseUnitTest.kt

```diff
@@ -0,0 +1,14 @@
+package io.customer.commontest
+
+import org.junit.After
+import org.junit.Before
+
+abstract class BaseUnitTest {
```

Comment:
I'm okay if we decide to remove this, but we can't yet use `BaseTest` in simple unit test due to context requirement there. So I added this so we can track simple unit tests when we improve test structure in future.